src/App.js

```diff
@@ -2,19 +2,27 @@ import styled from 'styled-components';
 import {Routes, Route} from 'react-router-dom';
 import FormPage from './pages/FormPage';
 import CalendarPage from './pages/CalendarPage';
+import NavTop from './components/NavTop';
+import NavBottom from './components/NavBottom';
+import {loadFromLocalStorage, writeToLocalStorage} from './util/localstorage';
+import JournalEntry from './components/JournalEntry';
 
 const StyledPage = styled.div`
-  padding: 15px;
+  padding: 35px 15px;
 `;
 
 export default function App() {
   return (
-    <StyledPage>
-      <Routes>
-        <Route path="/" element={<FormPage />} />
-
-        <Route path="calendarpage" element={<CalendarPage />} />
-      </Routes>
-    </StyledPage>
+    <>
+      <NavTop />
+      <StyledPage>
+        <Routes>
+          <Route path="/" element={<FormPage />} />
+          <Route path="calendarpage" element={<CalendarPage />} />
+          <Route path="journalentrypage" element={<JournalEntry />} />
+        </Routes>
+      </StyledPage>
+      <NavBottom />
```

Comment:
Some semantics here: I would suggest to use `main` element for the StyledPage instead of a `div` and a `header` and `footer`, respectively, for `NavTop` and `NavBottom`.